config/core/test-configs-chromeos.yaml

```diff
@@ -24,49 +24,60 @@ test_plans:
       <<: *chromeos-install-modules-params
       fixed_kernel: true
 
-  chromeos-tast: &chromeos-tast
-    pattern: 'chromeos/chromeos-tast-template.jinja2'
-    params: &chromeos-tast-params
-      job_timeout: '60'
-      prompt_command: 'sof-audio-pci'
-      docker_image: 'kernelci/chromeos-tast'
+  cros-boot: &cros-boot
+    pattern: 'chromeos/cros-boot.jinja2'
 
-  chromeos-tast-fixed:
-    <<: *chromeos-tast
+  cros-boot-fixed:
+    <<: *cros-boot
     params:
-      <<: *chromeos-tast-params
       fixed_kernel: true
 
-  chromeos-tast-qemu: &chromeos-tast-qemu
-    pattern: 'chromeos/chromeos-tast-qemu.jinja2'
-    filters:
-      - blocklist: &kselftest_defconfig_filter
+  cros-boot_qemu: &cros-boot_qemu
+    base_name: cros-boot
+    pattern: 'chromeos/cros-boot-qemu.jinja2'
+    filters: &cros-qemu-filters
+      - blocklist:
           defconfig: ['kselftest']
       - passlist:
           lab: ['lab-collabora-staging', 'lab-collabora']
-    params: &chromeos-tast-qemu-params
-      job_timeout: '60'
-      docker_image: 'kernelci/chromeos-tast'
-      rootfs_url: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/20220429.0/chromiumos-amd64-generic/amd64/'
 
-  chromeos-tast-qemu-fixed:
-    <<: *chromeos-tast-qemu
+  cros-boot_qemu-fixed:
+    <<: *cros-boot_qemu
+    base_name: cros-boot-fixed
     params:
-      <<: *chromeos-tast-qemu-params
       fixed_kernel: true
 
-  cros-boot: &cros-boot
-    pattern: 'chromeos/cros-boot.jinja2'
+  cros-flash:
+    pattern: 'chromeos/cros-flash.jinja2'
+    params:
+      job_timeout: '60'
 
-  cros-boot-fixed:
-    <<: *cros-boot
+  cros-tast: &cros-tast
+    pattern: 'chromeos/cros-tast.jinja2'
+    params: &cros-tast-params
+      job_timeout: '60'
+      docker_image: 'kernelci/cros-tast'
+
+  cros-tast-fixed:
+    <<: *cros-tast
     params:
+      <<: *cros-tast-params
       fixed_kernel: true
 
-  cros-flash:
-    pattern: 'chromeos/cros-flash.jinja2'
+  cros-tast_qemu: &cros-tast-qemu
+    <<: *cros-tast
+    base_name: cros-tast
+    filters: *cros-qemu-filters
+    params: &cros-tast-qemu-params
+      <<: *cros-tast-params
+      base_template: 'chromeos/cros-boot-qemu.jinja2'
+
+  cros-tast_qemu-fixed:
+    <<: *cros-tast-qemu
+    base_name: cros-tast-fixed
```

Comment:
This was missing in the first version of the PR so the `cros-tast-fixed` results ended up in the wrong place for QEMU. The next round of results should hopefully be OK with all the `-fixed` tests passing.